File: JavaScript/primo.js
function Primo(n) {
  if (n < 2) return false;

  for (let i = 2; i * i <= n; i++) {
    if (n % i === 0) return false;
  }
  return true;
}

const num = 9;

console.log("--- Verificador de Número Primo ---");
console.log("O número " + num + (Primo(num) ? " é primo" : " não é primo"));

File: Java/Primo.java
public class Primo {

    public static boolean ehPrimo(int n) {
        if (n < 2) return false;

        for (int i = 2; i * i <= n; i++) {
            if (n % i == 0) return false;
        }
        return true;
    }

    public static void main(String[] args) {
        int num = 9;

        System.out.println("--- Verificador de Numero Primo ---");
        System.out.println("O numero " + num + (ehPrimo(num) ? " e primo" : " nao e primo"));
    }
}